python/example_code/kms/kms_scenario.py

```diff
@@ -0,0 +1,361 @@
+                        "Action": "kms:*",
+                        "Resource": "*",
+                    }
+                ],
+            }
+
+            self.key_policy.set_new_policy(self.key_id, policy)
+            print("Key policy replacement succeeded.")
+            q.ask("Press Enter to continue...")
+        else:
+            print("Skipping replacing the key policy.")
+
+        print(DASHES)
+        print(f"12. Get the key policy\n")
+        print(
+            f"The next bit of code that runs gets the key policy to make sure it exists."
+        )
+        q.ask("Press Enter to continue...")
+        policy = self.key_policy.get_policy(self.key_id)
+        print(f"The key policy is: {policy}")
+
+        q.ask("Press Enter to continue...")
+        print(DASHES)
+        print(f"13. Create an asymmetric KMS key and sign your data\n")
+        print(
+            """
+        Signing your data with an AWS key can provide several benefits that make it an attractive option
+        for your data signing needs. By using an AWS KMS key, you can leverage the
+        security controls and compliance features provided by AWS,
+        which can help you meet various regulatory requirements and enhance the overall security posture
+        of your organization.
+        """
+        )
+        q.ask("Press Enter to continue...")
+        print(f"Sign and verify data operation succeeded.")
+        self.asymmetric_key_id = self.key_manager.create_asymmetric_key()
+        message = "Here is the message that will be digitally signed"
+        signature = self.key_encryption.sign(self.asymmetric_key_id, message)
+        if self.key_encryption.verify(self.asymmetric_key_id, message, signature):
+            print("Signature verification succeeded.")
+        else:
+            print("Signature verification failed.")
+
+        q.ask("Press Enter to continue...")
+        print(DASHES)
+        print(f"14. Tag your symmetric KMS Key\n")
+        print(
+            """
+        By using tags, you can improve the overall management, security, and governance of your
+        KMS keys, making it easier to organize, track, and control access to your encrypted data within
+        your AWS environment
+        """
+        )
+        q.ask("Press Enter to continue...")
+        self.key_manager.tag_resource(self.key_id, "Environment", "Production")
+        self.clean_up()
+
+    def is_key_enabled(self, key_id: str) -> bool:
+        """
+        Check if the key is enabled or not.
+
+        :param key_id: The key to check.
+        :return: True if the key is enabled, otherwise False.
+        """
+        response = self.key_manager.describe_key(key_id)
+        return response["Enabled"] is True
+
+    def clean_up(self):
+        if self.alias_name != "":
+            print(f"Deleting the alias {self.alias_name}.")
+            self.alias_manager.delete_alias(self.alias_name)
+        window = 7
```

Comment:
(nit) Magic number. I had to parse through the code for a minute to understand how window and `7` were related.

Reply:
I think this is a good suggestion.